dace/sdfg.py

```diff
@@ -3776,14 +3740,15 @@ def validate(self, sdfg, state_id) -> None:
             # If scope(src) contains scope(dst), then src must be a data node
             elif scope_contains_scope(scope, src_node, dst_node):
                 if not isinstance(src_node, nd.AccessNode):
-                    raise InvalidSDFGEdgeError(
-                        "Memlet creates an "
-                        "invalid path (source node %s should "
-                        "be a data node)" % str(src_node),
-                        sdfg,
-                        state_id,
-                        eid,
-                    )
+                    pass
```

Comment:
Why is this not an error any longer?

Reply:
Because we can connect a tasklet directly through a map. This removes the need for this valuation check and the test below